freva-rest/src/freva_rest/databrowser_api/core.py

```diff
@@ -658,16 +703,73 @@ def adjust_time_string(
             raise ValueError(f"Choose `time_select` from {methods}") from exc
         start, _, end = time.lower().partition("to")
         try:
-            start = parse(
-                start or "1", default=datetime(1, 1, 1, 0, 0, 0)
-            ).isoformat()
+            start = parse(start or "1", default=datetime(1, 1, 1, 0, 0, 0)).isoformat()
             end = parse(
                 end or "9999", default=datetime(9999, 12, 31, 23, 59, 59)
             ).isoformat()
         except ParserError as exc:
             raise ValueError(exc) from exc
         return [f"{{!field f=time op={solr_select}}}[{start} TO {end}]"]
 
+    @staticmethod
+    def adjust_bbox_string(
+        bbox: str,
+        bbox_select: str = "flexible",
+    ) -> List[str]:
+        """Adjust the bbox select keys to a solr spatial query using RPT
+        (Recursive Prefix Tree)
+
+        Parameters
+        ----------
+
+        bbox: str, default: ""
+            Special search facet to refine/subset search results by spatial extent.
+            This can be a string representation of a bounding box or a WKT polygon.
+            Valid strings are ``min_lon,max_lon by min_lat,max_lat`` for bounding
+            boxes and Well-Known Text (WKT) format for polygons. **Note**: Longitude
+            values must be between -180 and 180, latitude values between -90 and 90.
+        bbox_select: str, default: flexible
+            Operator that specifies how the spatial extent is selected. Choose from
+            flexible (default), strict or file. ``strict`` returns only those files
+            that fully contain the query extent. The bbox search ``-10,10 by -10,10``
+            will not select files covering only ``0,5 by 0,5`` with the ``strict``
+            method. ``flexible`` will select those files as it returns files that
+            have any overlap with the query extent. ``file`` will only return files
+            where the entire spatial extent is contained by the query geometry.
+
+        Raises
+        ------
+        ValueError: If parsing the coordinates failed or if bbox_select is invalid.
+        """
+        if not bbox:
+            return []
+        bbox = "".join(part.strip() for part in bbox.split())
```

Comment:
It would be better if the code would expect bbox to be a list or tuple for that matter of four entries e.g [0,50, 100, 120]